mlflow/server/js/src/experiment-tracking/components/CompareRunView.js

```diff
@@ -67,51 +96,90 @@ export class CompareRunView extends Component {
       <Link to={Routes.getExperimentPageRoute(experimentId)}>{experiment.getName()}</Link>,
       title,
     ];
+
+    window.addEventListener('resize', (e) => this.adjustTableColumnWidth(runInfos.length), true);
+    // adjust width immediately before loading page.
+    setImmediate(() => this.adjustTableColumnWidth(runInfos.length));
+
+    function onTableBlockScrollHanlder(e) {
+      const blocks = document.querySelectorAll('.compare-table .table-block');
+      for (let index = 0; index < blocks.length; ++index) {
+        const block = blocks[index];
+        if (block !== e.target) {
+          block.scrollLeft = e.target.scrollLeft;
+        }
+      }
+    }
```

Comment:
Why need this handler:

here we have several table blocks: “run_id/name/.. block”, “parameter block”, “metric block”,
when any of them scroll **horizontally**, then other block must scroll together.

previously, they are always scroll together because they are in the same table,

But, after we add the vertical scroll bar to “parameter block”, “metric block”, things changed, these blocks they scroll individually in horizon direction, when one scroll, others do not scroll.

So we need js code to control this, the js code triggerred by one DOM element “onscroll”, but it need control other DOM element scroll offset

Reply:
This recording show how horizontally scroll looks like
https://user-images.githubusercontent.com/19235986/151130077-dbf4bde3-c8d2-486b-aab7-e30210ff1aed.mov